packages/headless/src/controllers/commerce/core/facets/regular/headless-commerce-regular-facet.test.ts

```diff
@@ -9,24 +9,28 @@ import {buildMockCommerceRegularFacetResponse} from '../../../../../test/mock-co
 import {buildMockCommerceFacetSlice} from '../../../../../test/mock-commerce-facet-slice';
```

Comment:
Migrated/updated here: https://github.com/coveo/ui-kit/pull/3616/files#diff-7f0d585cbf7a88dfa49cd1527a2d3ad817e1105998ee0a77af220c3adc3be575

You can revert the changes you made to this file too if you want.